src/components/podcast/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import podcast from '../../assets/podcast.png';
+
+// var podcast = Podcast;
+
+const Podcast = () => (
+  <>
```

Comment:
Same here. I'm sure I had probably used it once in the template where it was unnecessary, but it's only to avoid having to use a div to return multiple siblings